client/context/broadcast.go

```diff
@@ -44,6 +80,10 @@ func (ctx CLIContext) BroadcastTxCommit(txBytes []byte) (sdk.TxResponse, error)
 
 	res, err := node.BroadcastTxCommit(txBytes)
 	if err != nil {
+		if errRes := CheckTendermintError(err, txBytes); errRes != nil {
+			return *errRes, err
```

Comment:
I don't think we actually want `err` returned here (and below) because the `TxResponse` won't be returned to the client.